Linux Script/configure-linux.sh

```diff
@@ -945,7 +948,7 @@ checkIfTLS()
 						esac
 			done			
 	    else
-		    logMsgToConfigSysLog "WARN" "WARN: Your system logs are being send insecurely. We prefer to send system logs securely so switching to secure configuration."
+		    logMsgToConfigSysLog "WARN" "WARN: Your system logs are being sent insecurely. We prefer to send the system logs securely so switching to the secure configuration."
```

Comment:
@Shwetajain148 what conditions cause this message to be displayed?

Reply:
@mostlyjason, This is the else part of the same **checkIfTLS** function and this message will be displayed if the suppress mode is on. It means whenever the user will run the script in **insecure** mode with the **suppress** flag then the script will not ask to switch to secure mode instead will copy the secure configuration by default.

Reply:
The problem seems unclear to me can you remove this change and put in a separate JIRA issue?